wear/src/main/java/io/homeassistant/companion/android/home/views/DetailsPanelView.kt

```diff
@@ -0,0 +1,163 @@
+@Composable
+fun DetailsPanelView(
+    entity: Entity<*>,
+    onBrightnessChanged: (Float) -> Unit,
+    onColorTempChanged: (Float) -> Unit
+) {
+    WearAppTheme {
+        ThemeLazyColumn {
+            val attributes = entity.attributes as Map<*, *>
+            item {
+                val friendlyName = attributes["friendly_name"].toString()
+                ListHeader(friendlyName)
+            }
+
+            if (entity.entityId.split('.')[0] == "light") {
+                // Brightness
+                // On HA Core 2021.5 and later brightness detection has changed
+                // to simplify things in the app lets use both methods for now
+                val supportedColorModes = attributes["supported_color_modes"] as? List<String>
+                val supportsBrightness =
+                    if (supportedColorModes == null) false else (supportedColorModes - NO_BRIGHTNESS_SUPPORT).isNotEmpty()
+                if (supportsBrightness || ((attributes["supported_features"] as Int) and SUPPORT_BRIGHTNESS_DEPR == SUPPORT_BRIGHTNESS_DEPR)) {
+                    val minValue = 0f
+                    val maxValue = 100f
+                    var currentValue =
+                        (attributes["brightness"] as? Number)?.toFloat()?.div(255f)?.times(100)
+                            ?: 0f
+                    if (currentValue < minValue)
+                        currentValue = minValue
+                    if (currentValue > maxValue)
+                        currentValue = maxValue
+
+                    item {
+                        Row(
+                            modifier = Modifier.fillMaxWidth().padding(horizontal = 8.dp)
+                        ) {
+                            Text(stringResource(R.string.brightness))
+                        }
+                    }
+                    item {
+                        InlineSlider(
+                            value = currentValue,
+                            onValueChange = { brightness ->
+                                onBrightnessChanged(brightness.div(100).times(255))
+                            },
+                            steps = 20,
+                            valueRange = minValue..maxValue,
+                            decreaseIcon = {
+                                Image(
+                                    asset = CommunityMaterial.Icon.cmd_brightness_4,
+                                    colorFilter = ColorFilter.tint(Color.White)
+                                )
+                            },
+                            increaseIcon = {
+                                Image(
+                                    asset = CommunityMaterial.Icon.cmd_brightness_7,
+                                    colorFilter = ColorFilter.tint(Color.White)
+                                )
+                            },
+                            modifier = Modifier.padding(bottom = 8.dp)
+                        )
+                    }
```

Comment:
For readability this should probably be changed in to a Composable function (also for color temperature)

Reply:
I tried to do this, but couldn't get that to work due to the `item`s. But now that I look at it again, I could put the `InlindleSlider` into it's own function. And maybe also separate the functionality checks into functions?

Reply:
`item` requires a `*LazyListScope` to work, so you might need to change the code a little to make it work.

> separate the functionality checks into functions

That is actually a really good idea, maybe you can use extension functions on the `Entity` class so they can also be re-used in other places?